tf2/src/cache.cpp

```diff
@@ -60,6 +60,8 @@ TimeCache::TimeCache(ros::Duration max_storage_time)
 : max_storage_time_(max_storage_time)
 {}
 
```

Comment:
namespace tf2 {
namespace cache {
...everything....
} 
}

is consistent with both the ROS C++ style guide and the Google C++ style guide, as well. So it's the way to go.